docs/dev_guide/biodata/buddies/inject.rst

```diff
@@ -0,0 +1,100 @@
+Injecting the Buddy Service
+=============================
+
+A typical use case for a Chado Buddy is in an importer.
+Best practice is to inject the buddy service into your importer class.
+Let's see how that is done.
+
+First you will need to include these classes in your importer:
+
+.. code::
+
+  use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+  use Symfony\Component\DependencyInjection\ContainerInterface;
+  use Drupal\tripal_chado\ChadoBuddy\PluginManagers\ChadoBuddyPluginManager;
```

Comment:
Include the actual buddies here as well to enable full type hinting and more clearly indicate dependencies.
```suggestion
  use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
  use Symfony\Component\DependencyInjection\ContainerInterface;
  use Drupal\tripal_chado\ChadoBuddy\PluginManagers\ChadoBuddyPluginManager;
  use Drupal\tripal_chado\Plugin\ChadoBuddy\‎ChadoDbxrefBuddy;
  use Drupal\tripal_chado\Plugin\ChadoBuddy\‎ChadoCvtermBuddy;
  use Drupal\tripal_chado\Plugin\ChadoBuddy\‎ChadoPropertyBuddy;
```